dvc/repo/add.py

```diff
@@ -1,18 +1,35 @@
 import os
+import logging
+import colorama
 
 from dvc.repo.scm_context import scm_context
 from dvc.stage import Stage
-from dvc.utils import walk_files
+from dvc.utils import walk_files, LARGE_DIR_SIZE
 from dvc.exceptions import RecursiveAddingWhileUsingFilename
 
 
+logger = logging.getLogger(__name__)
+
+
 @scm_context
 def add(repo, target, recursive=False, no_commit=False, fname=None):
     if recursive and fname:
         raise RecursiveAddingWhileUsingFilename()
 
     targets = _find_all_targets(repo, target, recursive)
 
+    if os.path.isdir(target) and len(targets) > LARGE_DIR_SIZE:
+        logger.warning(
+            "You are adding a large directory recursively,"
+            " consider tracking it as a whole instead.\n"
+            "{purple}HINT:{nc} Use {cyan}rm <directory>/*.dvc`{nc}"
```

Comment:
`rm dir/*.dvc` won't work if there are directories within directories :) Also, not windows-compatible. Maybe just mention that user needs to remove dvc files with words instead of specific commands?

Reply:
good one, @efiop :+1: 
I was also thinking about a `dvc remove --stages --recursive dir` or something like that. For now, I'll only modify the message.